flow-server/src/main/java/com/vaadin/flow/server/frontend/TaskGenerateBootstrap.java

```diff
@@ -39,8 +39,8 @@
  */
 public class TaskGenerateBootstrap extends AbstractTaskClientGenerator {
 
-    static final String DEV_TOOLS_IMPORT = String
-            .format("import '@vaadin/flow-frontend/vaadin-dev-tools.js';%n");
+    static final String DEV_TOOLS_IMPORT = String.format("import '"
+            + FrontendUtils.JAR_RESOURCES_IMPORT + "vaadin-dev-tools.js';%n");
```

Comment:
```suggestion
    static final String DEV_TOOLS_IMPORT = String.format("import '%svaadin-dev-tools.js';%n", FrontendUtils.JAR_RESOURCES_IMPORT );
```
As it's string format we should just use the format format.